### ISSUE TITLE
deno crashes when --config value isn't a correct tsconfig file path

### ISSUE THREAD

**Issue by KluVerKamp** [denoland/deno#2549]
``` 
$ deno --config worngtsconfigpath.json examples/basic/app.ts 
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:997:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by kitsonk** [comment]
Yeah, we can handle that "crash" better and surface something more meaningful, but exiting after displaying an error is the right behaviour IMO.

**Comment by viktor-ku** [comment]
I am working on it.

**Comment by bartlomieju** [comment]
Fixed in #2686 

CC @ry
